### ISSUE TITLE
[Bug] Docker builds fail to compile PDAL 2.7.2 from source, missing release assets

### ISSUE THREAD

**Issue by echoix** [OSGeo/grass#3940]
## Describe the bug

PDAL released 2.7.2 today.

Contrary to the other releases, that new release doesn't include the same assets that are used in build scripts. Thus, downloading with an url similar to `https://github.com/PDAL/PDAL/releases/download/${PDAL_VERSION}/PDAL-${PDAL_VERSION}-src.tar.gz` will fail.

## To reproduce
See a failing Docker build here https://github.com/OSGeo/grass/actions/runs/9719096265/job/26828288072#step:7:3154, from this PR https://github.com/OSGeo/grass/pull/3937

## Expected behavior

Have PDAL build correctly from source, either with another URL, or have the docker builds be ran before being merged.

## Additional context

Filed an issue in the PDAL repo here: https://github.com/PDAL/PDAL/issues/4432

**Comment by echoix** [comment]
Yes it was me that merged the failing PR that could be reverted if you think its too impactful (for now its red statuses and no new uploads for 2 of the 4 docker images). But our workflows should be able to handle this, without having to do much work in parallel for some PRs. So I just went with it, as its currently the "designed" way to handle changes in dockerfiles.

**Comment by neteler** [comment]
Seems to be fixed upstream in PDAL a few hours ago.

Edouard Choinière ***@***.***> schrieb am Sa., 29. Juni 2024,
00:38:

**Comment by echoix** [comment]
Now, only the ubuntu_wxgui build fails, as the image has PROJ 8.2.1 (`#8 82.88 Setting up libproj-dev:amd64 (8.2.1-1) ...`), while debian has 9.1.1 `#8 48.26 Setting up libproj-dev:amd64 (9.1.1-1+b1) ...`)

I know that the cmake file to find proj changed in 2.7.2, but they also relaxed from requiring PROJ 9.2+ to only PROJ 9.0+

**Comment by echoix** [comment]
See https://github.com/OSGeo/grass/pull/3946

**Comment by ninsbl** [comment]
Still relevant after #6481 or are other distros / OSes still affected? We may utilize the new build script in case?

**Comment by echoix** [comment]
> Still relevant after [#6481](https://github.com/OSGeo/grass/pull/6481) or are other distros / OSes still affected? We may utilize the new build script in case?

Ubuntu wxGUI still needs to use the script from #6481. I only updated the OS+apt package names in #6510.
